hydra-node/src/Hydra/Chain/Direct/Tx/Init.hs

```diff
@@ -0,0 +1,250 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Smart constructors for creating Hydra protocol transactions to be used in
+-- the 'Hydra.Chain.Direct' way of talking to the main-chain.
+--
+-- This module also encapsulates the transaction format used when talking to the
+-- cardano-node, which is currently different from the 'Hydra.Ledger.Cardano',
+-- thus we have not yet "reached" 'isomorphism'.
+module Hydra.Chain.Direct.Tx.Init where
+
+import Hydra.Cardano.Api
+import Hydra.Prelude
+
+import qualified Data.Set as Set
+import Hydra.Chain (HeadId (..), HeadParameters (..))
+import Hydra.Chain.Direct.Tx.Helpers (
+  InitialThreadOutput (..),
+  UTxOWithScript,
+  assetNameFromVerificationKey,
+  headValue,
+  hydraHeadV1AssetName, findFirst, mkHeadId,
+ )
+import Hydra.ContestationPeriod (ContestationPeriod, fromChain, toChain)
+import qualified Hydra.Contract.Head as Head
+import qualified Hydra.Contract.HeadState as Head
+import qualified Hydra.Contract.HeadTokens as HeadTokens
+import qualified Hydra.Contract.Initial as Initial
+import Hydra.Contract.MintAction (MintAction (Mint))
+import Hydra.Ledger.Cardano.Builder (
+  addOutputs,
+  addVkInputs,
+  emptyTxBody,
+  mintTokens,
+  unsafeBuildTransaction,
+ )
+import Hydra.Party (Party, partyFromChain, partyToChain)
+import Plutus.Orphans ()
+
+-- * Create Hydra Head transactions
```

Comment:
Should: use consistent haddocks in modules. Either remove these sections or have them in all individual tx modules.